src/util.ts

```diff
@@ -727,9 +737,36 @@ export class Util {
     config: MakeRequestConfig,
     callback: BodyResponseCallback
   ): void | Abortable {
+    const AUTO_RETRY_DEFAULT = true;
```

Comment:
Should these constants also reside at the class level with some comments / docs?

Reply:
Good callout!